compiler/src/dotty/tools/dotc/transform/ExpandPrivate.scala

```diff
@@ -95,17 +98,4 @@ class ExpandPrivate extends MiniPhaseTransform with IdentityDenotTransformer { t
     ensurePrivateAccessible(tree.symbol)
     tree
   }
-
-  override def transformDefDef(tree: DefDef)(implicit ctx: Context, info: TransformerInfo) = {
```

Comment:
This method only ever changed denotations for param accessors. The rest is handled by `transformIdent` and `transformSelect` above which I did not touch. If you merge your fix for #2054 I'll rebase to make sure everything still works.